commit 9db0b8d54024bf5a2d0c3239aa571d890e8c8b3a
Author: Eric Anholt <eric@anholt.net>
Date:   Fri Oct 23 12:33:43 2015 +0100

    drm/vc4: Remove extra barrier()s aroudn CTnCA/CTnEA setup.
    
    The writel() that these expand to already does barriers.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/drivers/gpu/drm/vc4/vc4_gem.c b/drivers/gpu/drm/vc4/vc4_gem.c
index eeb0925..0cea723 100644
--- a/drivers/gpu/drm/vc4/vc4_gem.c
+++ b/drivers/gpu/drm/vc4/vc4_gem.c
@@ -104,14 +104,11 @@ submit_cl(struct drm_device *dev, uint32_t thread, uint32_t start, uint32_t end)
 {
 	struct vc4_dev *vc4 = to_vc4_dev(dev);
 
-	V3D_WRITE(V3D_CTNCA(thread), start);
-	barrier();
-
-	/* Set the end address of the control list.  Writing this
-	 * register is what starts the job.
+	/* Set the current and end address of the control list.
+	 * Writing the end register is what starts the job.
 	 */
+	V3D_WRITE(V3D_CTNCA(thread), start);
 	V3D_WRITE(V3D_CTNEA(thread), end);
-	barrier();
 }
 
 int
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1098_9db0b8d54024bf5a2d0c3239aa571d890e8c8b3a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1098_9db0b8d54024bf5a2d0c3239aa571d890e8c8b3a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
